packages/ui/src/components/Modal/index.tsx

```diff
@@ -270,7 +270,7 @@ export const Modal = memo(
     animated = false,
     animation = 'zoom',
     ariaLabel = 'modal',
-    baseId = 'modal',
+    id,
```

Comment:
You also forgot to remove `unstable_xx` from props, it might be linked to typing of the Modal 😄 

<img width="241" alt="Screenshot 2023-03-08 at 14 19 02" src="https://user-images.githubusercontent.com/15812968/223723704-7b81f503-c718-46e3-a8b8-7fa1f729521e.png">